internal/hud/renderer.go

```diff
@@ -196,14 +196,9 @@ func isCrashing(res view.Resource) bool {
 }
 
 func bestLogs(res view.Resource) string {
-	// TODO(maia): get the BEST logs (currently getting any old logs)
-	if res.IsDC() {
-		lastBuildLog := strings.TrimSpace(string(res.LastBuild().Log))
-		if res.LastBuild().StartTime.Before(res.PodCreationTime) &&
-			len(lastBuildLog) > 0 {
-			return lastBuildLog + "\n\n" + res.PodLog
-		}
-		return res.PodLog
+	// TODO(maia): get the BEST logs (currently getting all logs)
```

Comment:
again, b/c we're not wrestling with pods that come and go, I think the "best" log for a dc service is just... it's log. I'll remove this comment.